AABB_tree/doc/AABB_tree/Concepts/AABBGeomTraits_2.h

```diff
@@ -0,0 +1,177 @@
+
+/*!
+\ingroup PkgAABBTreeConcepts
+\cgalConcept
+
+The concept `AABBGeomTraits_2` defines the requirements for the first template parameter of the class
+`CGAL::AABB_traits_2<AABBGeomTraits_2, AABBPrimitive>`. It provides predicates and constructors to detect
+and compute intersections between query objects and the primitives stored in the AABB tree.
+In addition, it contains predicates and constructors to compute distances between a point query
+and the primitives stored in the AABB tree.
+
+\cgalRefines{SearchGeomTraits_2}
+
+\cgalHasModelsBegin
+\cgalHasModelsBare{All models of the concept `Kernel`}
+\cgalHasModelsEnd
+
+\sa `CGAL::AABB_traits_2<AABBGeomTraits_2,AABBPrimitive>`
+\sa `CGAL::AABB_tree<AABBTraits>`
+\sa `AABBPrimitive`
+
+*/
+
+class AABBGeomTraits_2 {
+public:
+
+/// \name Types
+/// @{
+
+/*!
+A functor object to detect intersections between two geometric objects.
+Provides the following operators:
+
+`bool operator()(const Query& q, const Bbox_2& b)`,
+
+`bool operator()(const Query& q, const Primitive::Datum& d)`,
+
+`bool operator()(const Circle_2& c, const Bbox_2& b)`.
+
+The operator returns `true` iff there is an intersection.
+*/
+typedef unspecified_type Do_intersect_2;
+
+/*!
+A functor object to construct the intersection between two geometric objects.
+
+Provides the operator:
+
+`return_type operator()(const Query& q, const Primitive::Datum& d)`,
+
+which computes the intersection between `q` and `d`. The type of the returned object
+must be a `std::optional` of a `std::variant` of the possible intersection types.
+*/
+typedef unspecified_type Intersect_2;
+
+/*!
+A functor object to construct the circle centered at one point and passing through another one.
```

Comment:
"[...] and passing through another one."

does not match the API which is center + squared radius.

(Same for the 3D version.)